shared/chat/conversation/header/container.js.flow

```diff
@@ -1,10 +1,10 @@
 // @flow
 import {Component} from 'react'
 
-export type OwnProps = {
+export type OwnProps = {|
   onBack: () => void,
   onToggleSidePanel: () => void,
   sidePanelOpen: boolean,
-}
+|}
 
 export default class HeaderContainer extends Component<void, OwnProps, void> {}
```

Comment:
Made this and OwnType so its clear what the connected component expects as its own props so we can type check that too